posts/insitudiff/index.qmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "InSituDiff: perturbation analysis for spatial transcriptomics"
+author:
+  - name: Patrick Danaher
+    orcid: 0000-0002-2844-5883
+    affiliations:
+      - ref: nstg
+      - ref: patrickjdanaher
+date: "2024-10-29"
+date-modified: "2024-10-29"
+categories: [exploratory analysis]
+draft: false
+image: figures/insitudiff-tile.png
```

Comment:
Suggest adding `description: short phrase or description of the tool`
This will show up along with your figure on the blog view, rather than just showing the first few sentences of your post. See this post vs one with a short description: 
![{4E07B5ED-F20C-46B2-AB72-1D4B52F9E00F}](https://github.com/user-attachments/assets/6b4aecf3-38e6-4c89-adea-0c6a792b5ee7)  
![{E4435C8E-859F-45E6-A31A-94EB528F5B2B}](https://github.com/user-attachments/assets/c3fba281-ba49-45a8-9446-b0129f1cde08)